code/modules/organs/blood.dm

```diff
@@ -139,9 +151,10 @@ var/const/BLOOD_VOLUME_SURVIVE = 40
 						var/min_eff_damage = max(0, W.damage - 10) / 6 //still want a little bit to drip out, for effect
 						blood_max += max(min_eff_damage, W.damage - 30) / 30
```

Comment:
The 30s here use the `blood_loss_divisor`, if you're making that a var you should probably make them use it too in case it's changed to keep them the same.